### PR TITLE
fix #73350, #73352,

### LINKED ISSUE #73350: heise.de
### Issue URL (Annoyance)
 [https://www.heise.de/](https://adguardteam.github.io/AnonymousRedirect/redirect.html?url=https%3A%2F%2Fwww.heise.de%2F)
### Comment
Username: @Morku
> Cookie Pop-up
> --
> Die folgenden Informationen werden automatisch generiert. Bitte löschen Sie sie nicht.
> 
> Custom userscripts:
> https://userscripts.adtidy.org/release/adguard-extra/1.0/adguard-extra.user.js
> 

### Screenshots
<details>
  <summary>Screenshot 1</summary>

  Screenshot 1
</details>

### System configuration

Information | value
--- | ---
AdGuard product: | AdGuard for Windows v7.5.3
Browser: | Firefox
AdGuard driver: | WFP
Stealth mode options: | Hide your search queries, <br/>Send Do-Not-Track header, <br/>Remove X-Client-Data header from HTTP requests, <br/>Strip URLs from tracking parameters
Filters: | <b>Ad Blocking:</b><br/> AdGuard Mobile Ads, <br/>AdGuard Base<br/><br/><b>Privacy:</b><br/> AdGuard Tracking Protection<br/><br/><b>Social Widgets:</b><br/> AdGuard Social Media<br/><br/><b>Annoyances:</b><br/> AdGuard Annoyances<br/><br/><b>Language-specific:</b><br/> AdGuard Russian, <br/>AdGuard German

### PR DESCRIPTION
**Pull request by wout-codes**
#73350, #73352,

### CODE REVIEW

**Review by BlazDT — COMMENTED** [review]
Checked https://www.heise.de/news/Selbst-gemachter-Solar-Wasserstoff-ist-Stromquelle-fuer-zu-Hause-5035579.html which has a video and working when blocked. Did not find an article with social embeds yet.

**Comment by wout-codes** [comment]
@zloyden Can you check this? I am exhausted of Merging branch master.

File: AnnoyancesFilter/sections/cookies_specific.txt [hunk 1]
```diff
@@ -85,11 +85,13 @@ rtl.de,faz.net,chip.de,preis.de,giga.de,motorcyclenews.com,finanzen.net,vorname.
...
+/wrapperMessagingWithoutDetection.js$script,domain=heise.de|dpd.com|dpd.de|duden.de|sky.de|sky.com|tvmovie.de|cardscout.de|meineorte.com|tecchannel.de|itpro.co.uk|tvnow.at|tvnow.de|netdoktor.de|computerwoche.de|the-sun.com|thescottishsun.co.uk|thesun.co.uk|thesun.ie|handelsblatt.com|macwelt.de|lancashiretelegraph.co.uk|idealo.de|lvz.de|myhomebook.de|travelbook.de|4players.de|n-page.de|planet3ds.de|planetds.de|planetgameboy.de|planetswitch.de|planetvita.de|playstationportable.de|portablegaming.de|gala.de|denofgeek.com|classicfm.com|globalplayer.com|radio.pl|politico.eu|expertreviews.co.uk
 /wrapperMessagingWithoutDetection.*.js$script,domain=netzwelt.de
 ||cdn.privacy-mgmt.com^$important,domain=promipool.de|android-hilfe.de|wsj.com
 !
 !
+samsngexperts.ru##div[class^="modal n-modal in"]
+johnnyseeds.com##.grecaptcha-badge
```
Comment (zloyden) [inline AnnoyancesFilter/sections/cookies_specific.txt:94]: Why not `johnnyseeds.com##.cookie-bar-container`?

File: AnnoyancesFilter/sections/cookies_specific.txt [hunk 2]
```diff
@@ -2123,7 +2125,7 @@ britishcouncil.org.tr,britishcouncil.org###bc-cookie-banner
 thelifecoshop.com,tokgozler.com,flydolap.com,enplus.com.tr,dermosiparis.com,ambarexpress.com,cossta.com.tr,papscover.com,saten.com,modamizbir.com,deponda.com,yonkamuzikmarket.com,krisjoycosmetics.com,e-sansiro.com,cagri.com,bambi.com.tr,1001fidan.com,yolcudanalin.com.tr,eyupsabrituncer.com,platin.com.tr,dermokozmetik.com###cerekKullanimUyari
 allecodes.de###closeGdprContent
 computerweekly.com###cmp-banner
-dccomics.com###consent-modal
+greatist.com,dccomics.com###consent-modal
```
Comment (zloyden) [inline AnnoyancesFilter/sections/cookies_specific.txt:2128]: @AdguardTeam/filters-maintainers please check it on iOS.

File: AnnoyancesFilter/sections/cookies_specific.txt [hunk 3]
```diff
@@ -85,11 +85,13 @@ rtl.de,faz.net,chip.de,preis.de,giga.de,motorcyclenews.com,finanzen.net,vorname.
...
+/wrapperMessagingWithoutDetection.js$script,domain=heise.de|dpd.com|dpd.de|duden.de|sky.de|sky.com|tvmovie.de|cardscout.de|meineorte.com|tecchannel.de|itpro.co.uk|tvnow.at|tvnow.de|netdoktor.de|computerwoche.de|the-sun.com|thescottishsun.co.uk|thesun.co.uk|thesun.ie|handelsblatt.com|macwelt.de|lancashiretelegraph.co.uk|idealo.de|lvz.de|myhomebook.de|travelbook.de|4players.de|n-page.de|planet3ds.de|planetds.de|planetgameboy.de|planetswitch.de|planetvita.de|playstationportable.de|portablegaming.de|gala.de|denofgeek.com|classicfm.com|globalplayer.com|radio.pl|politico.eu|expertreviews.co.uk
 /wrapperMessagingWithoutDetection.*.js$script,domain=netzwelt.de
 ||cdn.privacy-mgmt.com^$important,domain=promipool.de|android-hilfe.de|wsj.com
 !
 !
+samsngexperts.ru##div[class^="modal n-modal in"]
```
Comment (zloyden) [inline AnnoyancesFilter/sections/cookies_specific.txt:93]: The website has low visitors. You should remove this rule.

**Comment by zloyden** [comment]
@wout-codes about `greatist.com,dccomics.com###consent-modal`. Please remove `greatist.com` from the rule, otherwise it disables scrolling of the page.

**Comment by wout-codes** [comment]
> @wout-codes about `greatist.com,dccomics.com###consent-modal`. Please remove `greatist.com` from the rule, otherwise it disables scrolling of the page.

Works fine for me.

**Comment by zloyden** [comment]
Hm, I checked `greatist.com` with another IP and your rule doesn't do anything. Btw, the cookie notice can't be hide on iOS.

**Review by zloyden — APPROVED** [review]

**Comment by zloyden** [comment]
I agree with other changes, nice one!

**Pull request merged**